markers/rhv.py

```diff
@@ -0,0 +1,23 @@
+"""rhv(tier_number): Run only tests marked with RHV tier, e.g. rhv2.
```

Comment:
this mark registration is incorrect, as in the tests you dont use `rhv(number)` but rather `"rhv{number}".format(number)`

so this would fail in pytest strict mode this would fail

in addition we already have tier selection via `fixtures/node_annotate.py`

and believe the combination of `--use-provider` and `--tier` in combination might be able to fit the outlined use-case

please get in touch for more details

Reply:
Regarding the first point, you are probably right. I wanted to use the marker with --strict, but pytest failed on another marker (i think it was "requires" marker) before it got to this one. I can of course fix that in accordance with your suggestion.

Regarding second point, I thought of that but I don't think it covers our use case. The reason is that some test case might be marked with highest --tier from the point of CFME QE team but in the context of our integration project, it might not be so important. Or vice versa.

Reply:
@jan-zmeskal thanks for the update, the requires marker is indeed our fault, please open an issue, we can take a look at fixing it but you can also just fix it while your are at it

wrt your own tier markers - it might help to replicate what we did fir the tier marker for the rhv tier in order to help with the selection

Reply:
@RonnyPfannschmidt thank you for the input, I'll take a look if I can get it more in line with how markers are implemented within the framework